atlas_densities/densities/utils.py

```diff
@@ -456,7 +456,7 @@ def get_aibs_region_names(region_map: "RegionMap") -> Set[str]:
 
 def get_hierarchy_info(
     region_map: "RegionMap",
-    root: str = "Basic cell groups and regions",
```

Comment:
This isn't the root name, was `Basic cell groups and regions` also changed?

Reply:
No, I just replaced it with its `acronym` as that's what the function is expecting now.

Reply:
But based on the PR description, the change is required (?) because the `name of the root region changed from "root" to "Whole Mouse Brain".`; so I'm confused as to why this change is required?

Reply:
Since the new code is [looking for](https://github.com/BlueBrain/atlas-densities/pull/42/files#diff-37b2b6d6dbbd641e4c91035d7a55d87f97811c43092505c492242a31c017ba37R485) the `acronym` attribute of a region rather than for the `name`, the "root" argument of this function must be the acronym now.

Reply:
I meant, why do we need to change to acronym from "name", when the only places that are changing in this PR weren't affected by the change of name?

Reply:
If we don't change the default value of this "root" argument (from the region name to the region acronym), when this function is called without the "root" argument it will not find any region because it will look for a region name among the regions [acronyms](https://github.com/BlueBrain/atlas-densities/pull/42/files#diff-37b2b6d6dbbd641e4c91035d7a55d87f97811c43092505c492242a31c017ba37R485).
How would you apply this change instead?

Reply:
But *why* are we changing from looking up by `acronym` instead of `name`; I don't understand the reasoning behind this change.

Reply:
Because it's a more robust search than with names, which can be long strings subject to variation more often than acronyms.

Reply:
I'd also like to change this, but this code can be used as a library, so changing fundamental API calls like this feels dangerous, so the bar for making changes is high.  Are you calling this somewhere where you got caught by the change in names?

Reply:
Yes, I ran one step of the main ReadMe with the new version of the hierarchy and got an empty result because there is no longer a region with name "root".
The usage of `acronym` already [exists](https://github.com/BlueBrain/atlas-densities/blob/16981b5d244d9cdde82ed6b098564f8296c2dbfa/atlas_densities/combination/annotations_combinator.py#L48).

Reply:
> one step of the main ReadMe 

Which step?  The examples in the README are meant to be run by external uses of the code and they don't have access to the internal hierarchy.

To maintain backwards compat, we could add an `attr` argument, which defaults to `name`, but can be overridden to be `acronym`, what do you think?

Reply:
> Which step? The examples in the README are meant to be run by external uses of the code and they don't have access to the internal hierarchy.

I ran [this step](https://github.com/BlueBrain/atlas-densities/blob/16981b5d244d9cdde82ed6b098564f8296c2dbfa/README.rst#fit-transfer-functions-from-mean-region-intensity-to-neuron-density) with the new hierarchy as input.

> To maintain backwards compat, we could add an `attr` argument, which defaults to `name`, but can be overridden to be `acronym`, what do you think?

That's an option. The overwriting will happen for [this call](https://github.com/BlueBrain/atlas-densities/blob/3b224e83f4eefa638c5f757ab3734a714459df76/atlas_densities/densities/fitting.py#L588) of `get_hierarchy_info`.
Shall I proceed?

Reply:
I have created https://github.com/BlueBrain/atlas-densities/pull/45 which allows one to choose the name of the region that is being fit. Thus, one should be able to use alternate hierarchies.